06-find.md

```diff
@@ -393,6 +393,34 @@ North Whitehead wrote in 1911, "Civilization advances by extending the
 number of important operations which we can perform without thinking
 about them."
 
+> ## Using grep {.challenge}
+>
+> ~~~
+> The Tao that is seen
+> Is not the true Tao, until
+> You bring fresh toner.
+>
+> With searching comes loss
+> and the presence of absence:
+> "My Thesis" not found.
+>
+> Yesterday it worked
+> Today it is not working
+> Software is like that.
+> ~~~
+>
+> From the above text, contained in the file haiku.txt, which command would result in the 
```

Comment:
Wrap "haiku.txt" with backtick quotes:

~~~
From the above text, contained in the file `haiku.txt`, which command would result in the
~~~